pkg/statistics/handle/storage/read.go

```diff
@@ -576,7 +576,15 @@ func loadNeededIndexHistograms(sctx sessionctx.Context, statsCache util.StatsCac
 		return nil
 	}
 	index, ok := tbl.Indices[idx.ID]
-	if !ok {
+	// Double check if the index is really needed to load.
```

Comment:
Or can you print out the index names that do not exist in stats_histogram records? This will be handy for later troubleshooting.

Reply:
We already have this log in other places.

Reply:
Updated some logs.